web/src/cypress/support/ep.ts

```diff
@@ -35,6 +36,7 @@ function createEP(ep?: EPOptions): Cypress.Chainable<EP> {
         name: ep.name || 'SM EP ' + c.word({ length: 8 }),
         description: ep.description || c.sentence(),
         repeat: ep.repeat || c.integer({ min: 1, max: 5 }),
+        favorite: ep.favorite,
```

Comment:
Fall back to random:
```suggestion
        favorite: ep.favorite ?? c.bool(),
```

Reply:
addressed